packages/dom-utils/package.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "@trezor/dom-utils",
+    "version": "1.0.0",
+    "private": true,
+    "license": "See LICENSE.md in repo root",
+    "sideEffects": false,
+    "main": "src/index",
+    "scripts": {
+        "lint": "eslint '**/*.{ts,tsx,js}'",
+        "test:unit": "jest -c ../../jest.config.base.js --passWithNoTests",
```

Comment:
remove test script please, there is very small chance to have test in future anyway